testsuite/do_tests.sh.in

```diff
@@ -517,6 +517,24 @@ echo "--------------------------"
 if test "x$(sli -c 'statusdict/have_mpi :: =')" = xtrue ; then
     junit_open 'core.phase_5'
 
+    # Check for old version of the /mpirun command, which had the NEST executable hardcoded
+    if test "x$(sli -c '/mpirun load cva_t Flatten length 3 eq =')" = xtrue ; then
```

Comment:
This test needs to be done before the first test if NEST is compiled with MPI, because all serial tests are then run as `mpirun -np 1` via `nest_serial`.